### PR TITLE
:sparkles: Add top channels

### PR DESCRIPTION
**Pull request by MatievisTheKat**
### This pull request adds two new features
- A leaderboard of top-chatted-in text channels
- The total amount of channels chatted in

The leaderboard only includes channels of guilds that the user is still a member of.
The total count includes channels of guilds the user isn't a member of anymore.

Demo:
ZJSMs4Bytf
image

### CODE REVIEW

File: src/views/Home.svelte [hunk 1]
```diff
@@ -60,6 +61,15 @@
...
+                    strokeWidth={1}
+                    strokeLinecap="square"
+                    strokeLinejoin="square"
+                    content="You have spoken in % different text channels"
+                    count="{ channels.length }"
+                    explanation="That's ~{ Math.round(channels.length / $data.guilds.length) } per guild!"
+                />
```
Comment (Androz2091) [inline src/views/Home.svelte:72]: This should be moved just below `You used X Slash Commands` 🤔

**Review by Androz2091 — CHANGES_REQUESTED** [review]
Thank you for your pull request 👍 
Some comments just before merging it

File: src/views/Home.svelte (hunk 1 above)
Comment (MatievisTheKat) [reply src/views/Home.svelte]: Done 👍

**Review by Androz2091 — APPROVED** [review]
Thank you for your contribution 👍

**Comment by Androz2091** [comment]
Uh, whoops, one last thing I forgot, you should also add demo data (in the app/store.js file) 😅 
and it would be nice if the "About" card was aligned to the left instead

**Comment by MatievisTheKat** [comment]
sure

File: src/app/store.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ if (isDemo) {
             isDM: true,
             messages: [demoMessageObject]
         }),
+        guilds: new Array(randomNumber(10, 100)).fill({ id: 123456789, name: "Demo" }),
```
Comment (Androz2091) [inline src/app/store.js:40]:
```suggestion
        guilds: new Array(randomNumber(10, 100)),
```

File: src/app/store.js [hunk 3]
```diff
@@ -35,9 +35,12 @@ if (isDemo) {
         user: demoUserObject,
         channels: new Array(randomNumber(200, 600)).fill({
             isDM: true,
+            guild: {
+                name: 'Awesome Discord Server'
+            },
             messages: [demoMessageObject]
         }),
-        guilds: new Array(randomNumber(10, 100)).fill({ id: 123456789, name: "Demo" }),
+        guilds: new Array(randomNumber(10, 100)).length,
```
Comment (MatievisTheKat) [inline src/app/store.js:46]: guilds is expected to be an array. the length property is used in the Home.svelte file

**Pull request merged**